src/status_im/utils/fx.cljs

```diff
@@ -51,11 +51,14 @@
     (swap! handlers/handler-nesting-level inc))
   (let [[first-arg & rest-args] args
         initial-fxs? (map? first-arg)
-        fx-fns (if initial-fxs? rest-args args)]
-    (clojure.core/reduce (fn [fxs fx-fn]
-                           (let [updated-cofx (update-db cofx fxs)]
-                             (if fx-fn
-                               (safe-merge fxs (fx-fn updated-cofx))
-                               fxs)))
-                         (if initial-fxs? first-arg {:db db})
-                         fx-fns)))
+        fx-fns (if initial-fxs? rest-args args)
+        res
+        (clojure.core/reduce (fn [fxs fx-fn]
+                               (let [updated-cofx (update-db cofx fxs)]
+                                 (if fx-fn
+                                   (safe-merge fxs (fx-fn updated-cofx))
+                                   fxs)))
+                             (if initial-fxs? first-arg {:db db})
+                             fx-fns)]
+    (swap! handlers/handler-nesting-level dec)
```

Comment:
could you please elaborate on this change ?

Reply:
This is completely unrelated to PR objective (if you ignore the fact that it helped me with debugging a bit), it just fixes a tiny issue with tracing fx handlers, `nasting-level` was not properly decremented. I can file a separate PR but I'm not sure it is really necessary.